Python/compile.c

```diff
@@ -1949,6 +1949,25 @@ compiler_default_arguments(struct compiler *c, arguments_ty args)
     return funcflags;
 }
 
+static int corrected_firstlineno(struct compiler *c, stmt_ty s,
+                                 asdl_seq * decos)
+{
+        /* To keep the ability to get the relevant source of a decorated item
+        using inspect.getsource, we need to keep the first line number
+        of the code object as the first line number of the first decorator.
+        This used to be done via the AST, but it is better to hide this
+        internally.
+        */
+        if (asdl_seq_LEN(decos) > 0) {
```

Comment:
`> 0` is not needed. Using just `asdl_seq_LEN(decos)` in the boolean context is common.